### PR TITLE
Pull test-infra scripts directly

### PR DESCRIPTION
**Pull request by coryrc**
## Description

This allows us to remove go.mod circular dependency for test-infra.

We aren't pulling any Go files from test-infra, so we never should
have been using dep or go mod to pull the scripts in.

TODO: a separate commit to remove knative.dev/test-infra from go.mod

### CODE REVIEW

File: scripts/test-infra/markdown-lint-config.rc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# For help, see
+# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```
Comment (mattmoor) [inline scripts/test-infra/markdown-lint-config.rc:2]: Remove trailing whitespace:
```suggestion
# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```

File: scripts/test-infra/performance-tests.sh [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  kubectl create secret generic mako-secrets \
+    --from-file=robot.json=${GOOGLE_APPLICATION_CREDENTIALS} \
+    --from-file=github-token=${GITHUB_TOKEN} \
+    --from-file=slack-read-token=${SLACK_READ_TOKEN} \
+    --from-file=slack-write-token=${SLACK_WRITE_TOKEN}
+  # Delete all benchmark jobs to avoid noise in the update process
+  echo ">> Deleting all cronjobs and jobs on cluster $1 in zone $2"
+  kubectl delete cronjob --all
+  kubectl delete job --all
+
```
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:74]: Remove trailing whitespace:
```suggestion

```

File: scripts/test-infra/performance-tests.sh [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+# Try to reconcile clusters for benchmarks in the current repo.
+# This function will be run as postsubmit jobs.
+function reconcile_benchmark_clusters() {
+  header "Reconciling clusters for ${REPO_NAME}"
+  run_perf_cluster_tool --reconcile \
+    --gcp-project=${PROJECT_NAME} --repository=${REPO_NAME} --benchmark-root=${BENCHMARK_ROOT_PATH} \
+    || abort "failed reconciling clusters for ${REPO_NAME}"
+  header "Done reconciling clusters"
+  # For now, do nothing after reconciling the clusters, and the next update_clusters job will automatically
```
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:134]: Remove trailing whitespace:
```suggestion
  # For now, do nothing after reconciling the clusters, and the next update_clusters job will automatically
```

File: scripts/test-infra/performance-tests.sh [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+# Parse flags and excute the command.
+function main() {
+  if (( ! IS_PROW )); then
+    abort "this script should only be run by Prow since it needs secrets created on Prow cluster"
+  fi
+
+  # Set up the user credential for cluster operations
+  setup_user || abort "failed to set up user"
+
+  # Try parsing the first flag as a command.
```
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:148]: Remove trailing whitespace:
```suggestion
  # Try parsing the first flag as a command.
```

**Comment by coryrc** [comment]
PR out to fix the build issue (missing copyright): https://github.com/knative/test-infra/pull/2067

File: scripts/test-infra/dummy.go [hunk 5]
```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor) [inline scripts/test-infra/dummy.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: scripts/test-infra/markdown-lint-config.rc (hunk 1 above)
Comment (mattmoor) [inline scripts/test-infra/markdown-lint-config.rc:2]: Remove trailing whitespace:
```suggestion
# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```

File: scripts/test-infra/performance-tests.sh (hunk 2 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:74]: Remove trailing whitespace:
```suggestion

```

File: scripts/test-infra/performance-tests.sh (hunk 3 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:134]: Remove trailing whitespace:
```suggestion
  # For now, do nothing after reconciling the clusters, and the next update_clusters job will automatically
```

File: scripts/test-infra/performance-tests.sh (hunk 4 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:148]: Remove trailing whitespace:
```suggestion
  # Try parsing the first flag as a command.
```

File: scripts/test-infra/dummy.go (hunk 5 above)
Comment (mattmoor) [inline scripts/test-infra/dummy.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: scripts/test-infra/markdown-lint-config.rc (hunk 1 above)
Comment (mattmoor) [inline scripts/test-infra/markdown-lint-config.rc:2]: Remove trailing whitespace:
```suggestion
# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```

File: scripts/test-infra/performance-tests.sh (hunk 2 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:74]: Remove trailing whitespace:
```suggestion

```

File: scripts/test-infra/performance-tests.sh (hunk 3 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:134]: Remove trailing whitespace:
```suggestion
  # For now, do nothing after reconciling the clusters, and the next update_clusters job will automatically
```

File: scripts/test-infra/performance-tests.sh (hunk 4 above)
Comment (mattmoor) [inline scripts/test-infra/performance-tests.sh:148]: Remove trailing whitespace:
```suggestion
  # Try parsing the first flag as a command.
```

**Comment by coryrc** [comment]
abandoning for a clean start.

**Pull request closed without merging**